### PR TITLE
PVW-7: Setup landing image

### PR DESCRIPTION
**Pull request by filipomarcellino**

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -1,29 +1,63 @@
-/* RegoReg60 */
```
Comment (Julian-UBC) [inline client/src/App.css:1]: Can you keep this comment there

File: client/src/App.css [hunk 2]
```diff
@@ -1,29 +1,63 @@
...
-  font-family: 'RegoReg', Arial;
+  font-family: "RegoReg", Arial;
   font-size: 3.75rem;
   font-style: normal;
   font-weight: 400;
   line-height: normal;
   letter-spacing: -0.075rem;
 }
 
-/* RegoReg48 */
```
Comment (Julian-UBC) [inline client/src/App.css:11]: This one too

File: client/src/App.css [hunk 3]
```diff
@@ -1,29 +1,63 @@
...
-  font-family: 'RegoReg', Arial;
+  font-family: "RegoReg", Arial;
   font-size: 3rem;
   font-style: normal;
   font-weight: 400;
   line-height: normal;
   letter-spacing: -0.06rem;
 }
 
-/* RegoReg30 */
```
Comment (Julian-UBC) [inline client/src/App.css:21]: This one too

File: client/src/App.css [hunk 4]
```diff
@@ -1,29 +1,63 @@
...
-  font-family: 'RegoReg', Arial;
+  font-family: "RegoReg", Arial;
   font-size: 1.875rem;
   font-style: normal;
   font-weight: 400;
   line-height: normal;
   letter-spacing: -0.0375rem;
 }
+
+.App-logo {
```
Comment (Julian-UBC) [inline client/src/App.css:28]: can u remove unnecessary css

File: client/src/pages/home/Home.tsx
Comment (Julian-UBC) [inline client/src/pages/home/Home.tsx:1]: Can you move your landing image to a component in the home

**Review by Julian-UBC — CHANGES_REQUESTED** [review]

File: client/src/App.css (hunk 4 above)
Comment (filipomarcellino) [reply client/src/App.css:31]: is the .app-logo not used?

File: client/src/App.css (hunk 2 above)
Comment (filipomarcellino) [reply client/src/App.css:11]: added

File: client/src/App.css (hunk 4 above)
Comment (Julian-UBC) [reply client/src/App.css:31]: Is it used by your landing image? If not, then it is not used

**Review by Julian-UBC — CHANGES_REQUESTED** [review]
Can you change `z-10` to `z-20` on line 41 in NavigationBar.tsx

File: client/src/shared/components/LandingImage.tsx
Comment (Julian-UBC) [inline client/src/shared/components/LandingImage.tsx:1]: Can you move this component to pages/home/components

File: client/src/shared/components/LandingImage.tsx [hunk 5]
```diff
@@ -0,0 +1,147 @@
+import { useCallback, useEffect, useRef, useState } from "react";
+import img2 from "../../assets/DSC01469.JPG";
+import img3 from "../../assets/DSC01479.JPG";
+import img4 from "../../assets/DSC01487.JPG";
+import img5 from "../../assets/DSC01610.JPG";
+import img6 from "../../assets/DSC01614.JPG";
+import img7 from "../../assets/DSC01481.JPG";
+import React from "react";
```
Comment (Julian-UBC) [inline client/src/shared/components/LandingImage.tsx:8]: you can remove this line 
```suggestion
```

File: client/src/shared/components/LandingImage.tsx [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+    const imageSizeRelative = 1 / 5;
+    top = Math.min(
+        Math.max(top, navbarSize * 100),
+        100 - imageSizeRelative * 100
+    );
+
+    return { top: `${top}%`, left: `${left}%` };
+  };
+
+  const [isVisible, setIsVisible] = useState(false);
```
Comment (Julian-UBC) [inline client/src/shared/components/LandingImage.tsx:80]: what is this state for?

**Review by Julian-UBC — CHANGES_REQUESTED** [review]

File: client/src/shared/components/LandingImage.tsx (hunk 6 above)
Comment (filipomarcellino) [reply client/src/shared/components/LandingImage.tsx:87]: it's for the fading effect on images and header

File: client/src/shared/components/LandingImage.tsx
Comment (filipomarcellino) [reply client/src/shared/components/LandingImage.tsx:1]: done

File: client/src/shared/components/LandingImage.tsx (hunk 5 above)
Comment (filipomarcellino) [reply client/src/shared/components/LandingImage.tsx]: done

File: client/src/App.css [hunk 7]
```diff
@@ -1,43 +1,33 @@
-/* H1: RegoBook60 */
+/* RegoReg60 */
```
Comment (Julian-UBC) [inline client/src/App.css:1]: I think you accidentally decline upcoming changes here

**Review by Julian-UBC — CHANGES_REQUESTED** [review]

**Review by Julian-UBC — APPROVED** [review]
Great work!!!

File: client/src/pages/home/components/LandingImageSmall.tsx [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+            >
+                <div className="z-20 font-Avenir flex flex-col items-start">
+                <span className="text-[27px] mdVertical:text-[48px] lgVertical:text-[63px] z-20">This is</span>
+                <span className="text-[27px] mdVertical:text-[48px] lgVertical:text-[63px] z-20 bg-[#102713] text-white px-1 rounded-sm">PERMIKA</span>
+                <span className="text-[27px] mdVertical:text-[48px] lgVertical:text-[63px] z-20 bg-[#102713] text-white px-1 mt-1 rounded-sm">Vancouver</span>
+        </div>
+
+
+
+                <p className="z-20 text-md mdVertical:text-2xl text-left w-9/12 text-gray-500 pt-4 font-RegoReg">
```
Comment (Julian-UBC) [inline client/src/pages/home/components/LandingImageSmall.tsx:83]:
```suggestion
                <p className="z-20 text-md mdVertical:text-2xl text-left w-9/12 text-[#9A9A9A] pt-4 font-RegoReg">
```

**Review by Julian-UBC — CHANGES_REQUESTED** [review]

**Review by mayaramadhina — APPROVED** [review]

**Review by Julian-UBC — APPROVED** [review]
Great work @filipomarcellino!!!

**Pull request merged**